### PR TITLE
[MRG] Raw psd as topography.

### PR DESCRIPTION
**Pull request by jaeilepp**
Closes https://github.com/mne-tools/mne-python/issues/2553.

The problem with IAS channels in the beginning of data makes using ``iter_topography`` for custom plots really annoying, but because the function does exactly as the docs say, I didn't want to change it. Instead I made a function for plotting the psd of raw data as topography.

### CODE REVIEW

File: mne/viz/raw.py [hunk 1]
```diff
@@ -670,3 +671,69 @@ def _plot_raw_traces(params, inds, color, bad_color, event_lines=None,
...
+        Number of jobs to run in parallel.
+    verbose : bool, str, int, or None
+        If not None, override default verbose level (see mne.verbose).
+
+    Returns
+    -------
+    fig : instance of matplotlib figure
+        Figure distributing one image per channel across sensor topography.
+    """
+    picks = pick_types(raw.info, meg=True, eeg=True, ref_meg=True, seeg=True,
```
Comment (agramfort) [inline mne/viz/raw.py]: can you reuse the common pick function for data channels?

Comment (jaeilepp) [inline mne/viz/raw.py]: The PR isn't merged yet, and I found the picks isn't even necessary here.

Comment (agramfort) [inline mne/viz/raw.py]: k

set the PR to MRG when ready.

File: mne/io/base.py [hunk 2]
```diff
@@ -1431,7 +1431,7 @@ def plot_psd(self, tmin=0.0, tmax=60.0, fmin=0, fmax=np.inf,
         Returns
         -------
         fig : instance of matplotlib figure
-            Figure distributing one image per channel across sensor topography.
```
Comment (jaeilepp) [inline mne/io/base.py]: This was just wrong...

**Comment by jaeilepp** [comment]
Minimal gist for trying this out: https://gist.github.com/jaeilepp/92b80ea3a0098a56d061

**Comment by agramfort** [comment]
thx !

**Pull request merged**